### PR TITLE
Remove logger config

### PR DESCRIPTION
**Pull request by pbell97**
Removing logger config project and the few references to it. Adding it to the imodeljs-internal repo in a separate PR.

### CODE REVIEW

**Comment by calebmshafer** [comment]
@Mergifyio update

File: docs/learning/common/Logging.md [hunk 1]
```diff
@@ -73,7 +71,7 @@ This approach is useful when you want to monitor the activity in a small number
...
+A service typically initializes and configures logging in its startup logic. Configuration can be based on the configuration parameters of the service, which may be set by the deployment mechanism. The simplest way to do this is to use [LoggerLevelsConfig]($bentleyjs-core). This is normally used in conjunction with a stream config.. Both are normally stored in a config file that is deployed with the service. Finally, some specific configuration parameters are normally set at runtime, based on deployment parameters using [EnvMacroSubst]($bentleyjs-core) to do macro substitution.
```
Comment (calebmshafer) [inline docs/learning/common/Logging.md:74]: Duplicate `.` at the end of the second sentence.

**Review by calebmshafer — CHANGES_REQUESTED** [review]
A minor comment on the docs update, otherwise looks good.

**Review by pmconne — APPROVED** [review]

**Review by calebmshafer — APPROVED** [review]

**Review by DanEastBentley — APPROVED** [review]

**Comment by calebmshafer** [comment]
@Mergifyio update

**Pull request merged**